node/cli/src/chain_spec.rs

```diff
@@ -216,6 +216,7 @@ pub fn testnet_genesis(
 	initial_authorities: Vec<(AccountId, AccountId, AuthorityId)>,
 	root_key: AccountId,
 	endowed_accounts: Option<Vec<AccountId>>,
+	enable_contract_debug: bool,
```

Comment:
When I discussed this a while ago we would have called this more concretely like `enable_println` or something like that. But I am not sure what is the better choice so leave it as it is for now.

Reply:
Yeah I was going back and forth between being more specific or general.

Reply:
While I agree with @Robbepop , I wouldn't block the PR on this.

Reply:
I agree too. Fixed